justfile

```diff
@@ -128,6 +126,8 @@ export INGESTION_PY_VERSION := `just ingestion_server/py-version`
 export FRONTEND_NODE_VERSION := `just frontend/node-version`
 export FRONTEND_PNPM_VERSION := `just frontend/pnpm-version`
 
+export HOST_NETWORK_ADDRESS := if os() == "macos" { "host.docker.internal" } else { "172.17.0.1" }
```

Comment:
Why is this variable moved?

Reply:
It's only used by the Docker Compose, so I placed it in under the Docker section with other variables of the same nature.